src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -153,9 +153,14 @@
       <FunctionalTest Include="$(TestArchivesRoot)*.zip" />
       <FunctionalTest Include="$(AnyOSTestArchivesRoot)*.zip" />
     </ItemGroup>
+
+    <PropertyGroup>
+        <_DesktopParameter Condition="'$(TargetGroup)' == 'net46'">--xunit-test-type=desktop</_DesktopParameter>
```

Comment:
This should probably have been appended to $(OtherRunnerScriptArgs) which seems to already exist.